src/Shipment/TaxIdentificationNumbers/TaxIdentificationNumbers.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Shipment\TaxIdentificationNumbers;
+
+use ArrayObject;
+use function array_map;
+
+class TaxIdentificationNumbers extends ArrayObject
+{
+    public function __construct(TaxIdentificationNumber ...$taxIdentificationNumbers)
+    {
+        parent::__construct($taxIdentificationNumbers);
+    }
+
+    public function toArray(): array
+    {
+        return array_map(static fn(TaxIdentificationNumber $taxIdentificationNumber) => $taxIdentificationNumber->toArray(), (array) $this);
+    }
```

Comment:
This function is not tested. While it is mocked a bit further down the line, I think it'd be best if we check if the real `TaxIdentificationNumberCollection::toArray()` will return an expected result for a set of one or more `TaxIdentificationNumber` types (I say types because when testing `TaxIdentificationNumberCollection` we can actually mock the `TaxIdentificationNumber` classes - they are not in the focus of a unit test for `TaxIdentificationNumberCollection`